src/bundler/bundle_v2.zig

```diff
@@ -9027,11 +9025,18 @@ const LinkerContext = struct {
                 chunk.template.placeholder.hash = chunk.isolated_hash;
 
                 const rel_path = std.fmt.allocPrint(c.allocator, "{any}", .{chunk.template}) catch unreachable;
+                bun.path.platformToPosixInPlace(rel_path);
                 if ((try path_names_map.getOrPut(rel_path)).found_existing) {
                     try c.log.addErrorFmt(null, Logger.Loc.Empty, bun.default_allocator, "Multiple files share the same output path: {s}", .{rel_path});
                     return error.DuplicateOutputPath;
                 }
-
+                // resolve any /./ and /../ occurrences
+                // use resolvePosix since we asserted above all seps are '/'
+                if (std.mem.indexOf(u8, rel_path, "/./") != null) {
+                    chunk.final_rel_path = std.fs.path.resolvePosix(c.allocator, &.{rel_path}) catch unreachable;
```

Comment:
Please use Bun’s path resolve function instead of Zig’s. Ours handles many more cases than theirs and is more well tested.